### ISSUE TITLE
Error 404 connecting to api.jsbin.com

### ISSUE THREAD

**Issue by o-alquimista** [jsbin/jsbin#3359]
I can no longer access my bin at: https://jsbin.com/xulahutumo/edit?output

On the console, this message is printed out:
```
failed to fetch from bin api - check network panel
```

And checking the requests, I can see it is failing to connect to `api.jsbin.com`.

![Captura de tela de 2020-01-20 17-06-49](https://user-images.githubusercontent.com/25424263/72754728-6814eb00-3ba7-11ea-81eb-99f28eb5544f.png)

When I open `api.jsbin.com` on the browser, it returns this:

![Captura de tela de 2020-01-20 17-13-19](https://user-images.githubusercontent.com/25424263/72755034-449e7000-3ba8-11ea-8f18-5e6fa098be56.png)

That was a huge bug reproducer that had to remain available for this [bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1610083). Please tell me I didn't lose that bin. I promise I will make a backup of it! :D

Oh, man... I'm so screwed... I think.
![Captura de tela de 2020-01-20 17-16-49](https://user-images.githubusercontent.com/25424263/72755187-abbc2480-3ba8-11ea-922a-a820bb919bff.png)

**Comment by o-alquimista** [comment]
The correct URL is now: https://jsbin.com/xulahutumo

With the update to v5, the old URL stopped working. Phew! That was close. I just went ahead and backed up everything in the bin.

Closing issue...

**Comment by o-alquimista** [comment]
Oh, actually, it's my fault. I switched to v5 in this browser session, so it sticked to it. The old URL still opens my bin in v4, if I open it in another browser or in private browsing.

**Comment by remy** [comment]
not quite sure I follow the last comment, but should I re-open?

**Comment by o-alquimista** [comment]
> not quite sure I follow the last comment, but should I re-open?

No, I mean there is no problem at all.

There's a button somewhere that lets us switch between v4 and v5. Well, I switched to v5 and forgot to switch back.

The session kept me on v5. It was only when I wiped all the active JS Bin sessions on the browser that it finally reset to v4. The URL for v5 is slightly different, that's why it gave me the impression that my bin was missing.
